Release v2.1.0-beta.1 (#3557)
mapbox/mapbox-navigation-ios: CHANGELOG.md

### CarPlay

* Added the `CarPlayActivity.panningInNavigationMode` case, which allows to track a state when user is panning a map view while actively navigating. ([#3545](https://github.com/mapbox/mapbox-navigation-ios/pull/3545))

> * Fix issue, which was causing panning dismissal button to stop working on CarPlay. ([#3543](https://github.com/mapbox/mapbox-navigation-ios/pull/3543))

Reviewer: We use past tense in changelogs:

```suggestion
* Fixed an issue that caused the panning dismissal button to stop working on CarPlay. ([#3543](https://github.com/mapbox/mapbox-navigation-ios/pull/3543))
```
